code/modules/mob/living/simple_mob/subtypes/animal/alien animals/teppi.dm

```diff
@@ -0,0 +1,1195 @@
+			wantpet = 0
+			petcount += 1
+			if(prob(20))
+				to_chat(M, "<span class='notice'>\The [src] grumbles at your touch.</span>")
+		else if(lets_eat(M) && prob(50))
+			to_chat(M, "<span class='notice'>\The [src] grumbles a bit... and then bowls you over, pressing their weight into yours to knock you off of your feet! In a rush of chaotic presses and schlorps, the gooey touch of Teppi flesh grinds over you as you're guzzled away! Casually swallowed down in retaliation for all of the pettings. Pumped down deep into the grumbling depths of \the [src].</span>")
+			visible_message("<span class='danger'>\The [src] scromfs \the [M], before chuffing and settling down again.</span>")
+			playsound(src, pick(bodyfall_sound), 75, 1)
+			teppi_pounce(M)
+			wantpet = 100
+	else
+		return ..()
+
+/mob/living/simple_mob/vore/alienanimals/teppi/examine()
+	. = ..()
+	if(item_type)
+		. += "<span class='notice'>They are wearing a [item_type] with [name] written on it.</span>"
+	if(nutrition >= 1000)
+		. += "<span class='notice'>They look well fed.</span>"
+	if(nutrition <= 500)
+		. += "<span class='notice'>They look hungry.</span>"
+	if(health < maxHealth && health / maxHealth * 100 <= 75)
+		. += "<span class='notice'>They look beat up.</span>"
+	
+
+/mob/living/simple_mob/vore/alienanimals/teppi/update_icon()
+	..()
+	teppi_icon()
+	if(ghostjoin)
+		ghostjoin_icon()
+
+
+/mob/living/simple_mob/vore/alienanimals/teppi/Life()
+	. =..()
+	if(!.)
+		return
+	wantpet += rand(0,2) * affection_factor
+	amount_grown += rand(1,5)
+	var/not_hungy = FALSE
+	if(nutrition >= 500)
+		not_hungy = TRUE
+	if(amount_grown >= 1000)
+		if(teppi_adult)
+			if(not_hungy && !teppi_wool)
+				nutrition -= rand(250,500)
+				teppi_wool = TRUE
+				breedable = TRUE
+				meat_amount += rand(0,2)
+				update_icon()
+		else if (not_hungy)
+			var/nutrition_cost = 500 + (nutrition / 2)
+			adjust_nutrition(-nutrition_cost)
+			new /mob/living/simple_mob/vore/alienanimals/teppi(loc, src)
+			qdel(src)
+		else 
+			visible_message("\The [src] whines pathetically...", runemessage = "whines")
+			if(prob(50))
+				playsound(src, 'sound/voice/teppi/whine1.ogg', 75, 1)
+			else
+				playsound(src, 'sound/voice/teppi/whine2.ogg', 75, 1)
+			amount_grown -= rand(100,250)
+	do_breeding(not_hungy)
+	do_healing(not_hungy)
+	if(prob(0.5))
+		teppi_sound()
+
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/do_healing(not_hungy)
+	if(!not_hungy)
+		return
+	if(health < maxHealth)
+		if(heal_countdown > 0)
+			heal_countdown -= 1
+			return
+		if(bruteloss > 0)
+			adjustBruteLoss(-2)
+		else if(fireloss > 0)
+			adjustFireLoss(-0.5)
+		nutrition -= 5
+		heal_countdown = 5
+
+/mob/living/simple_mob/vore/alienanimals/teppi/proc/do_breeding(not_hungy)
+	if(!breedable || !not_hungy || prevent_breeding)
+		return
+	if(client)	//Player controlled teppi get a verb, so just do the countdown
+		if(baby_countdown > 0)
+			baby_countdown -= 1
+		return
+	if(baby_countdown > 0)
+		baby_countdown -= 1
```

Comment:
Comment: You can just do `baby_countdown--` which subtracts one, and is *stylish*.

You can also do this on one line with `baby_countdown = max(baby_countdown-1, 0)` which sets it to the higher of baby_countdown-1 or 0 (which would be higher if it became a negative number). Your way might be faster though, I'm not really sure.